wiki/Ranking_Criteria/osu!catch/en.md

```diff
@@ -16,18 +16,19 @@ These terms are frequently appearing when getting in touch with osu!catch diffic
-- **Drop:** A medium-sized object represented by a slider tick.
-- **Droplet:** A small object representing a slider body. Missing these will reduce your accuracy, but unlike the above, will not result in a combo break.
+- **Fruit:** A large object representing a hitcircle, slider head, tail or repeat.
+- **Drop:** A medium-sized object representing a slider tick.
+- **Droplet:** A small object representing a slider body. Missing these will reduce your accuracy, but unlike fruits and drops, will not result in a combo break.
 - **Banana:** An object found during spinners. These award bonus points, but do not contribute to accuracy and are not required to obtain max combo.
-- **Jump:** A spacing between two objects that requires the use of dash to catch both.
-- **Hyperjump:** A more exaggerated spacing which cannot be caught by normal dashing. During play, hyperdash will be triggered between the two objects, characterised by a coloured outline on the first object.
-- **Edge dash:** A very large spacing between two objects where the required Trigger Distance is not reached, and as such, a hyperjump is not generated. The first object must be caught with the edge of the plate in order to catch the second object at all.
-- **Trigger distance:** The minimum spacing between two objects at which a hyperdash is generated between them.
-- **Antiflow:** A strong direction or velocity change that goes against a player's natural movement pattern.
-- **Basic dash/hyperdash:** Any dash or hyperdash whose time between fruits is at least twice the time required to allow dashes or hyperdashes, respectively. As an example, a hyperdash between objects separated by 250 ms in a Platter classifies as a basic hyperdash.
+- **Active object:** An object that must be caught in order to maintain combo, namely fruits and drops. 
+- **Walk:** Any spacing between two objects where no dash is needed to catch both.
+- **Dash:** A spacing between two objects that requires the use of the dash key to catch both.
+- **Hyperdash:** Objects generated when the spacing between two active objects is too far apart to be caught by normal dashing. During play, this is characterized by a coloured outline on the first object. The term is also used to describe the movement between the two objects. 
+- **Trigger distance:** The minimum spacing between two active objects at which a hyperdash is generated between them.
+- **Edge dash:** A very large spacing between two active objects where the required Trigger Distance is not reached, and as such, a hyperdash is not generated. The first object must be caught with the edge of the plate in order to catch the second object at all.
+- **Basic dash/hyperdash:** Any dash or hyperdash whose time between active objects is at least twice the time required to allow dashes or hyperdashes, respectively. As an example, a hyperdash between objects separated by 250 ms in a Platter classifies as a basic hyperdash.
```

Comment:
wouldn't this mean that edge dashes are a subset of basic dashes? is that intentional?

Reply:
No. Basic and higher-snapped dashes are defined based on the gap between notes in the timeline, whereas edge dashes are defined by the spacing in the playfield in relation to the trigger distance. An edge dash could be either basic or higher-snapped.

Reply:
makes sense, I found it confusing since "edge dash" and "basic dash" are defined next to each other but to catch players ig there is no confusion